app/templating/summary/question.py

```diff
@@ -67,11 +80,11 @@ def _build_checkbox_answers(cls, answer, answer_schema):
         multiple_answers = []
         for option in answer_schema['options']:
             if option['value'] in answer:
-                if option['value'] == 'other':
+                if option['value'].lower() == 'other':
                     summary_option_display_value = cls._get_checkbox_other_display_value(answer, answer_schema, option)
-                    multiple_answers.append(summary_option_display_value)
+                    multiple_answers.append((summary_option_display_value, True))
```

Comment:
Agree that dict would provide better clarity, but Yogi and I tried this last week and a simple dict did not guarantee ordering of answers on Summary screen. So we also tried using an ordered dict, but we ran into some other problem (cannot remember what it was now sorry - perhaps Yogi remembers?